pkg/apis/migration/migration.go

```diff
@@ -90,6 +91,8 @@ const (
 	CRDSingular = "cnsvspherevolumemigration"
 	// CRDPlural represent the plural name of cnsvspherevolumemigrations CRD
 	CRDPlural = "cnsvspherevolumemigrations"
+	// default interval for cnsvspherevolumemigrations crd cleanup
+	defaultCRDCleanupIntervalInMin = 5
```

Comment:
Can you make this configurable?

Reply:
leftover CRDs are not that harmful. Can we increase interval to 1 time a day to do cleanup?
QueryVolume is an expensive operation and should not be called un-nessarility every 5 minutes.

Reply:
+1
Changes can be found here: https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/440

I would like to keep the scope of this PR to just delete CR when PV delete is invoked for PVC with Reclaim policy.